### PR TITLE
MCLOUD-6806: Add possibility to access Magento on port 443 inside of …

### PR DESCRIPTION
**Pull request by shiftedreality**
…network

### Description

Port 443 (https://magento2.docker) is not accessible from other containers and it makes it not usable for CLI operations like MFTF tests.

*AC*
* Make port 443 accessible from all CLI containers

### Fixed Issues (if relevant)

1. https://jira.corp.magento.com/browse/MCLOUD-6806

### Manual testing scenarios

1. Build images
1. Deploy containers with `--with-selenium`
1. Install Magento
1. Run `docker-compose run --rm test curl -k http://magento2.docker`
1. Magento page returned
1. Run `docker-compose run --rm test curl -k https://magento2.docker`
1. Magento page returned

### Release notes

For user-facing changes, add a meaningful release note. For examples, see [Magento Cloud Docker release notes](https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html).

### Associated documentation updates

Add link to Magento DevDocs PR or Issue, if needed.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] Pull request introduces user-facing changes and includes meaningful release notes and documentation
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: src/Command/BuildCompose.php [hunk 1]
```diff
@@ -234,13 +234,18 @@ protected function configure(): void
         )->addOption(
             Source\CliSource::OPTION_HOST,
             null,
-            InputOption::VALUE_OPTIONAL,
+            InputOption::VALUE_REQUIRED,
```
Comment (oshmyheliuk) [inline src/Command/BuildCompose.php:237]: Why some options become required? can't we use default values?

Comment (oshmyheliuk) [reply src/Command/BuildCompose.php:237]: It looks like BIC changes and it can break some auto tools which are using this command.

Comment (shiftedreality) [reply src/Command/BuildCompose.php:237]: `InputOption::VALUE_OPTIONAL` means that you able to use `--host` without the value.

This is actually a bug, `InputOption::VALUE_REQUIRED` means that value is required in option

Comment (oshmyheliuk) [reply src/Command/BuildCompose.php:237]: Ah, I see, so that's should be ok.

**Review by oshmyheliuk — APPROVED** [review]

**Review by NadiyaS — APPROVED** [review]

**Review by arhiopterecs — APPROVED** [review]

**Comment by andriyShevtsov** [comment]
QA approved

**Pull request merged**